src/dev/src/styles.scss

```diff
@@ -1,5 +1,5 @@
+@import '../../clr-angular/color/variables.color';
```

Comment:
I just noticed but it seems like this file is missing the copyright notice. Right?

Reply:
Yup